jupyterlab_git/tests/test_detailed_log.py

```diff
@@ -13,7 +13,7 @@ async def test_detailed_log():
     with patch("jupyterlab_git.git.execute") as mock_execute:
         # Given
         process_output = [
-            "f29660a (HEAD, origin/feature) Commit message",
+            "    Test Description with leading and trailing spaces    ",
             "10\t3\tnotebook_without_spaces.ipynb",
```

Comment:
Let's add a return line character as it is present in the git output.

```suggestion
            "\n10\t3\tnotebook_without_spaces.ipynb",
```

Reply:
Hey @fcollonval,

I noticed that when we add the `"\n10\t3\tnotebook_without_spaces.ipynb"` that the return does not actually strip the `\n` at this point in time; thus, failing the test.

Would I want to modify the `strip_and_split` function to also strip out new lines?

Before:
`return s.strip("\x00").split("\x00")`
After:
`return s.strip("\x00").strip("\n").split("\x00")`

Reply:
Good suggestion @iflinda let's do that.